frost/Cargo.toml

```diff
@@ -0,0 +1,115 @@
+[package]
+name = "snarkvm-frost"
+version = "0.7.5"
+authors = [ "The Aleo Team <hello@aleo.org>" ]
+description = "Implementation of FROST - Flexible Round-Optimised Threshold Signature"
+homepage = "https://aleo.org"
+repository = "https://github.com/AleoHQ/snarkVM"
+keywords = [
+  "aleo",
+  "cryptography",
+  "blockchain",
+  "decentralized",
+  "zero-knowledge"
+]
+categories = [ "compilers", "cryptography", "mathematics", "wasm", "web-programming" ]
+include = [ "Cargo.toml", "src", "README.md", "LICENSE.md" ]
+license = "GPL-3.0"
+edition = "2021"
+
+[dependencies.snarkvm-curves]
+path = "../curves"
+version = "0.7.5"
+default-features = false
+
+[dependencies.console]
+package = "snarkvm-console"
+path = "../console"
+version = "0.7.5"
+features = ["algorithms"]
+
+[dependencies.snarkvm-fields]
+path = "../fields"
+version = "0.7.5"
+default-features = false
+
+[dependencies.snarkvm-parameters]
+path = "../parameters"
+version = "0.7.5"
+optional = true
+default-features = false
+
+[dependencies.snarkvm-r1cs]
+path = "../r1cs"
+version = "0.7.5"
+default-features = false
+
+[dependencies.snarkvm-utilities]
+path = "../utilities"
+version = "0.7.5"
+default-features = false
+
+[dependencies.anyhow]
+version = "1.0"
+
+[dependencies.bincode]
+version = "1"
+
+[dependencies.derivative]
+version = "2"
+features = [ "use_core" ]
+
+[dependencies.digest]
+version = "0.10"
+
+[dependencies.getrandom]
+version = "0.2"
+optional = true
+features = [ "js" ]
+default-features = false
+
+[dependencies.hashbrown]
```

Comment:
is this dependency needed? the standard library already uses it internally for `HashMap` and `HashSet`